modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -284,6 +294,14 @@ export class Wallet implements IWallet {
     return this._wallet.coin;
   }
 
+  type(): WalletType | undefined {
+    return this._wallet.type;
```

Comment:
We can probably default to 'hot' when this value is undefined.

Reply:
that's not gonna match the typing on the class.